src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/RadioButton.xaml

```diff
@@ -150,6 +150,7 @@
                                 <Condition Property="IsEnabled" Value="False" />
                             </MultiTrigger.Conditions>
                             <Setter Property="Foreground" Value="{DynamicResource RadioButtonForegroundDisabled}" />
+                            <Setter Property="Background" Value="{DynamicResource RadioButtonBackgroundDisabled}" />
```

Comment:
@anjali-wpf do we need this given that the radio button is not selected?

Reply:
yes, let us say you are disabling all radio buttons, in that case we would require this